src/components/Footer.js

```diff
@@ -0,0 +1,12 @@
+import React from 'react';
+import './Footer.scss';
+
+function Footer() {
+    return (
+        <footer className="Footer">
```

Comment:
The footer does not need a footer class because it's the only footer component on the page.